packages/esm-commons-lib/src/constants.ts

```diff
@@ -12,200 +10,5 @@ export const encounterRepresentation =
-export const hivCD4Result_UUID = '657AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const hivCD4Count_UUID = '5497AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-
-//HIV Death
-export const hivDeathDate_UUID = '1543AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const causeOFDeath_UUID = 'ef973f1f-557f-4620-acf5-9c4c18bf1eda';
-export const deathSpecific_UUID = 'e329cdf4-4eeb-4821-85ec-80ec4b503de0';
-export const deathFormEncounterType_UUID = '111c2104-991d-4b58-a30e-ce84bb275534';
-
-//Transfer Out
-export const transferOutEncounterType_UUID = '3044916a-7e5f-478b-9091-803233f27f91';
-export const visitDate_UUID = '163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const receivingFacility_UUID = '162724AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const TransferOutDate_UUID = '160649AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const verified_UUID = '797e0073-1f3f-46b1-8b1a-8cdad134d2b3';
-
-//Patient Tracing
-export const ContactDate_UUID = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const ContactMethod_UUID = '59c023dd-eed2-4b11-8c34-b88e9439db3c';
-export const ContactOutcome_UUID = 'bc45edbd-11e7-4888-ad7d-4ec3dd8cdcf6';
-export const PatientTracingEncounterType_UUID = '0cd5d4cb-204e-419a-9dd7-1e18e939ce4c';
-
-//Intimate Partner
-export const IntimatePartnerEncounterType_UUID = '881fff34-b4a9-4d11-b2f5-a8a23a9f402b';
-export const ThreatenedToHurt_UUID = 'bd86f7ee-1d5f-4f5d-aa0f-4680aa6e65cb';
-export const SexuallyMolested_UUID = '1246AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const howOftenThreatened_UUID = '953f6271-57ef-414e-bdba-fe9e0246db58';
-export const howOftenSexuallyMolested_UUID = '1dd53a22-2e8f-425b-8ba4-59172ed3fafe';
-
-//Contact Tracing
-export const ContactTracingEncounterType_UUID = '570e9e42-4306-41dc-9bf8-634bbc70a524';
-export const ContactTracingDate_UUID = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const ContactTracingMethod_UUID = '59c023dd-eed2-4b11-8c34-b88e9439db3c';
-export const ContactTracingOutcome_UUID = '36a3e671-9d60-4109-b41f-046f44f4b389';
-
-//Service Delivery
-export const ServiceDeliveryEncounterType_UUID = '62ee5922-a229-48d3-a1da-875c1ffa9436';
-export const CommunityDSDModel_UUID = '52824cbe-0e4d-4c18-8179-80b5799f34ed';
-
-//Peads Disclosure
-export const PeadsDisclosureEncounterType_UUID = '390c2f21-c1c4-4246-94ca-a026157cd1db';
-export const DisclosureDate_UUID = '85fbdcc8-8dbc-40a9-b85f-5d1bfe1ab63d';
-export const DisclosureStage_UUID = '573f93e2-12f6-483e-aa6e-14e9b76b311a';
-
-//Patner Notification
-export const PatnerNotificationEncounterType_UUID = '4dd0ee63-805f-43e9-833c-6386ba97fdc1';
-export const IndexHIVStatus_UUID = '1436AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const PatnerNotificationContactDate_UUID = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const FirstName_UUID = '166102AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
-export const Relationship_UUID = '85d3b4fe-c1a9-4e27-a86b-dcc1e30c8a93';
+export const BASE_WS_API_URL = '/ws/rest/v1/';
+export  const BASE_FHIR_API_URL = '/ws/fhir2/R4/';
```

Comment:
new line please, also these baseUrls might be available in the framework we don't need to redefine them

Reply:
Actually, this endpoint is not in use anymore